docs/development/vendoring.md is out of date

**1. What `kops` version are you running? The command `kops version`, will display
 this information.**

Master branch

**2. What Kubernetes version are you running? `kubectl version` will print the
 version if a cluster is running or provide the Kubernetes version specified as
 a `kops` flag.**

Not applicable

**3. What cloud provider are you using?**

Not applicable

**4. What commands did you run?  What is the simplest way to reproduce this issue?**

Read `docs/development/vendoring.md` section on "Adding a dependency to the vendor directory". Followed steps (1) and (2)

**5. What happened after the commands executed?**

```
echo "`make dep-ensure` has been replaced by `make gomod`"
/bin/sh: fork: Resource temporarily unavailable
/bin/sh: fork: Resource temporarily unavailable
/bin/sh: fork: Resource temporarily unavailable
/bin/sh: line 0: cd: /src/k8s.io/kops: No such file or directory
/bin/sh: fork: Resource temporarily unavailable
/bin/sh: fork: Resource temporarily unavailable
/bin/sh: fork: Resource temporarily unavailable
/bin/sh: fork: Resource temporarily unavailable
make[164]: *** [dep-ensure] Error 1
```
and so on.

**6. What did you expect to happen?**

The dependency to be imported and added to the `vendor` subdirectory, as per the documentation.

**7. Please provide your cluster manifest. Execute
  `kops get --name my.example.com -o yaml` to display your cluster manifest.
  You may want to remove your cluster name and other sensitive information.**

Not applicable

**8. Please run the commands with most verbose logging by adding the `-v 10` flag.
  Paste the logs into this report, or in a gist and provide the gist link here.**

```
dr33vp2:kops jgmyers$ make -v 10 dep-ensure
GNU Make 3.81
Copyright (C) 2006  Free Software Foundation, Inc.
This is free software; see the source for copying conditions.
There is NO warranty; not even for MERCHANTABILITY or FITNESS FOR A
PARTICULAR PURPOSE.

This program built for i386-apple-darwin11.3.0
```

**9. Anything else do we need to know?**

```
iMac:kops jgmyers$ ps x | grep 'make'
46305 s000  S+     0:00.02 /Library/Developer/CommandLineTools/usr/bin/make dep-ensure
46335 s000  S+     0:00.00 /bin/sh -c echo "`make dep-ensure` has been replaced by `make gomod`"
46336 s000  S+     0:00.00 /bin/sh -c echo "`make dep-ensure` has been replaced by `make gomod`"
46337 s000  S+     0:00.01 /Library/Developer/CommandLineTools/usr/bin/make dep-ensure
46367 s000  S+     0:00.00 /bin/sh -c echo "`make dep-ensure` has been replaced by `make gomod`"
46368 s000  S+     0:00.00 /bin/sh -c echo "`make dep-ensure` has been replaced by `make gomod`"
46369 s000  S+     0:00.01 /Library/Developer/CommandLineTools/usr/bin/make dep-ensure
46399 s000  S+     0:00.00 /bin/sh -c echo "`make dep-ensure` has been replaced by `make gomod`"
46400 s000  S+     0:00.00 /bin/sh -c echo "`make dep-ensure` has been replaced by `make gomod`"
46401 s000  S+     0:00.01 /Library/Developer/CommandLineTools/usr/bin/make dep-ensure
46431 s000  S+     0:00.00 /bin/sh -c echo "`make dep-ensure` has been replaced by `make gomod`"
46432 s000  S+     0:00.00 /bin/sh -c echo "`make dep-ensure` has been replaced by `make gomod`"
46433 s000  S+     0:00.01 /Library/Developer/CommandLineTools/usr/bin/make dep-ensure
46463 s000  S+     0:00.00 /bin/sh -c echo "`make dep-ensure` has been replaced by `make gomod`"
46464 s000  S+     0:00.00 /bin/sh -c echo "`make dep-ensure` has been replaced by `make gomod`"
46465 s000  S+     0:00.01 /Library/Developer/CommandLineTools/usr/bin/make dep-ensure
```
...and so on.

--- a/docs/development/vendoring.md
+++ b/docs/development/vendoring.md
@@ -15,28 +15,26 @@ update commands:
 * [go mod](https://github.com/golang/go/wiki/Modules) for kops 1.15 and newer branches (including master)
 * [hg](https://www.mercurial-scm.org/wiki/Download)
 
-<!-- TODO: update dependency management for go mod -->
-
 ## Adding a dependency to the vendor directory
 
-The `dep` tool will manage required dependencies based on the imports
+Go modules will manage required dependencies based on the imports
 found in the source code. Follow these steps to run the update process:
 
 1. Add the desired import to a `.go` file.
-1. Run `make dep-ensure` to start the update process. If this step is
+2. Run `make gomod` to start the update process. If this step is
 successful, the imported dependency will be added to the `vendor`
 subdirectory.
 1. Commit any changes, including changes to the `vendor` directory,
-`Gopkg.lock` and `Gopkg.toml`.
+`go.mod`, and `go.sum`.
 1. Open a pull request with these changes separately from other work
 so that it is easier to review.
 
 ## Updating a dependency in the vendor directory (e.g. aws-sdk-go)
 
-1. Update the locked version as specified in Gopkg.toml
-1. Run `make dep-ensure`.
-1. Review the changes to ensure that they are as intended / trustworthy.
-1. Commit any changes, including changes to the `vendor` directory,
-`Gopkg.lock` and `Gopkg.toml`.
+1. Update the locked version as specified in `go.mod`
+2. Run `make gomod`.
+3. Review the changes to ensure that they are as intended / trustworthy.
+4. Commit any changes, including changes to the `vendor` directory,
+`go.mod` and `go.sum`.
 1. Open a pull request with these changes separately from other work so that it
 is easier to review.  Please include any significant changes you observed.

